numpy/core/tests/test_ufunc.py

```diff
@@ -493,6 +493,11 @@ def test_sum_complex(self):
             d += d
             assert_almost_equal(d, 2. + 2j)
 
+    def test_sum_initializer(self):
+        assert_equal(np.sum([3], initializer=2), 5)
+        assert_almost_equal(np.sum(np.ones((2, 3, 5)), axis=(0, 2), initializer=2),
```

Comment:
Why `almost_equal` instead of `equal`? This should be exact also for float.

Reply:
I tend to keep all floating-point stuff `almost_equal`. It's just a habit.

Reply:
It's a nitpick, but I'd like this to be `equal` - it should be exact so we might as well test it as such.